engine/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProvider.java

```diff
@@ -160,7 +161,18 @@ public void initCounters() {
     @CacheResult(cacheName = "rbac-recipient-users-provider-get-group-users")
     public List<User> getGroupUsers(String accountId, boolean adminOnly, UUID groupId) {
         Timer.Sample getGroupUsersTotalTimer = Timer.start(meterRegistry);
-        RbacGroup rbacGroup = retryOnRbacError(() -> rbacServiceToService.getGroup(accountId, groupId));
+        RbacGroup rbacGroup;
+        try {
+            rbacGroup = retryOnRbacError(() -> rbacServiceToService.getGroup(accountId, groupId));
+        } catch (ClientWebApplicationException exception) {
+            // The group does not exist (or no longer exists - ignore)
+            if (exception.getResponse().getStatus() == 404) {
```

Comment:
Maybe a bit too scared, but I would maybe flip the left and right part of the equals.
We could also replace the 404 with a constant. I think we don't have to as close to everyone knows what 404 is, but replacing 404 with something "readable" like "not found" would improve readability.

This would do the same: 

if (HTTP_NOT_FOUND == exception.getResponse().getStatus()) {
    return List.of();
}

You can decide if you want to replace it or not. I am fine with both :-)

Reply:
PS: the import is from java.net.HttpURLConnection.

Reply:
I think the order is fine, but will add the NOT_FOUND constant

Reply:
Yea, the flip would save us if the exception object is null or the getResponse() returns null. But this probably never gonna happen.